x-pack/solutions/security/plugins/security_solution/public/detections/components/alert_summary/table/table.test.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import type { DataView } from '@kbn/data-views-plugin/common';
+import { createStubDataView } from '@kbn/data-views-plugin/common/data_views/data_view.stub';
+import { TestProviders } from '../../../../common/mock';
+import { Table } from './table';
+
+const dataView: DataView = createStubDataView({ spec: {} });
+
+describe('<Table />', () => {
+  it('should render all components', () => {
+    const { getByTestId } = render(
+      <TestProviders>
+        <Table dataView={dataView} groupingFilters={[]} />
+      </TestProviders>
+    );
+
+    expect(getByTestId('alertsTableErrorPrompt')).toBeInTheDocument();
```

Comment:
Isn't this the opposite of what the test says? I expect an error prompt means all components weren't rendered?

Reply:
No, the error prompt is coming from the alerts table component when no data is displayed. This is just to check that the alerts table is rendered. I did not want to create a complex integration tests and decided to stick with the most basic unit test here... I don't see the value in testing the way the alerts table works, this should be done within the alerts table code.

Reply:
👍🏾 Agree...I just wish it was called `emptyPrompt` not `errorPrompt`